src/precice/tests/ToolingTests.cpp

```diff
@@ -48,5 +48,27 @@ BOOST_AUTO_TEST_CASE(DTDReference)
   }
 }
 
+BOOST_AUTO_TEST_SUITE(ConfigCheck)
+
+BOOST_AUTO_TEST_CASE(Serial)
+{
+  BOOST_REQUIRE_NO_THROW(
+      precice::tooling::checkConfiguration(
+          precice::testing::getPathToSources() + "/precice/tests/config-checker.xml",
+          "SolverTwo",
+          1));
+}
+
+BOOST_AUTO_TEST_CASE(Parallel)
+{
+  BOOST_REQUIRE_NO_THROW(
+      precice::tooling::checkConfiguration(
+          precice::testing::getPathToSources() + "/precice/tests/config-checker.xml",
+          "SolverTwo",
+          4));
```

Comment:
Mostly a question than a comment: Is it intended that this test and the one in `CTestConfig.cmake` use a different communicator size?

Reply:
It is not intended, but also doesn't matter. From the preCICE perspective there are only 2 cases: 1 and >1